app/views/consultations/show.html.erb

```diff
@@ -35,8 +35,7 @@
 
           <%= render partial: "document_summary", locals: { document: @document } %>
         </div>
-      <% else %>
-
+      <% elsif @document.closed? %>
```

Comment:
I've switched it to something more explicit:
`<% elsif @document.not_yet_open? && @document.opening_at.present? %>`

I don't want to do too much spelunking.